packages/neon-core/src/wallet/components/AssetBalance.ts

```diff
@@ -20,15 +24,18 @@ export class AssetBalance implements NeonObject {
   public unspent: Coin[];
   public spent: Coin[];
   public unconfirmed: Coin[];
+  public hash: string;
 
-  public constructor(abLike: Partial<AssetBalanceLike> = {}) {
+  public constructor(abLike: AssetBalanceWithHash = { hash: "nonHash" }) {

```

Comment:
This kinds of defeats the purpose of the type system. If we are filling in a non-hash string for the user, it just means that the field itself is not guaranteed. Why not just use the old Partial<AssetBalanceLike> and throw an error if the hash field is empty.

Reply:
it a good suggestion, i will do this